apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/PreAdjFissClaimResponseTransformerV2.java

```diff
@@ -0,0 +1,150 @@
+package gov.cms.bfd.server.war.r4.providers;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import com.newrelic.api.agent.Trace;
+import gov.cms.bfd.model.rda.PreAdjFissClaim;
+import gov.cms.bfd.sharedutils.exceptions.BadCodeMonkeyException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.hl7.fhir.r4.model.ClaimResponse;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Meta;
+import org.hl7.fhir.r4.model.Reference;
+
+/** Transforms FISS/MCS instances into FHIR {@link ClaimResponse} resources. */
+public class PreAdjFissClaimResponseTransformerV2 {
+
+  private static final String METRIC_NAME =
+      MetricRegistry.name(PreAdjFissClaimResponseTransformerV2.class.getSimpleName(), "transform");
+
+  private static final Map<Character, String> STATUS_TEXT;
+
+  static {
+    STATUS_TEXT = new HashMap<>();
+    STATUS_TEXT.put('a', "Active");
+    STATUS_TEXT.put('s', "Suspend");
+    STATUS_TEXT.put('p', "Paid");
+    STATUS_TEXT.put('d', "Denied");
+    STATUS_TEXT.put('i', "Inactive");
+    STATUS_TEXT.put('r', "Reject");
+    STATUS_TEXT.put('t', "RTP");
+    STATUS_TEXT.put('m', "Move");
+  }
```

Comment:
Assuming this is meant to be immutable this should be a Guava ImmutableMap rather than a HashMap.